Custom build failing

I just upgraded from 4.2.0 to 4.3.2 and my custom build is no longer working. I get this error:
```
ERR! Trouble generating info: ERROR: Unable to find the plugin "../../config/jsdoc/info/api-plugin"
ERROR: Unable to find the plugin "../../config/jsdoc/info/define-plugin"
ERROR: Unable to find the plugin "../../config/jsdoc/info/virtual-plugin"
FATAL: Unable to load template: Cannot find module '../../config/jsdoc/info/publish'
```

--- a/config/jsdoc/api/conf.json
+++ b/config/jsdoc/api/conf.json
@@ -1,7 +1,7 @@
 {
     "opts": {
         "recurse": true,
-        "template": "../../config/jsdoc/api/template"
+        "template": "config/jsdoc/api/template"
     },
     "tags": {
         "allowUnknownTags": true
@@ -17,11 +17,11 @@
     },
     "plugins": [
         "plugins/markdown",
-        "../../config/jsdoc/api/plugins/inheritdoc",
-        "../../config/jsdoc/api/plugins/typedefs",
-        "../../config/jsdoc/api/plugins/events",
-        "../../config/jsdoc/api/plugins/observable",
-        "../../config/jsdoc/api/plugins/api"
+        "config/jsdoc/api/plugins/inheritdoc",
+        "config/jsdoc/api/plugins/typedefs",
+        "config/jsdoc/api/plugins/events",
+        "config/jsdoc/api/plugins/observable",
+        "config/jsdoc/api/plugins/api"
     ],
     "markdown": {
         "parser": "gfm"

--- a/config/jsdoc/info/conf.json
+++ b/config/jsdoc/info/conf.json
@@ -1,7 +1,7 @@
 {
     "opts": {
         "recurse": true,
-        "template": "../../config/jsdoc/info"
+        "template": "config/jsdoc/info"
     },
     "tags": {
         "allowUnknownTags": true
@@ -10,8 +10,8 @@
         "includePattern": "\\.js$"
     },
     "plugins": [
-        "../../config/jsdoc/info/api-plugin",
-        "../../config/jsdoc/info/define-plugin",
-        "../../config/jsdoc/info/virtual-plugin"
+        "config/jsdoc/info/api-plugin",
+        "config/jsdoc/info/define-plugin",
+        "config/jsdoc/info/virtual-plugin"
     ]
 }

